includes/OutputPage.php

```diff
@@ -797,8 +797,10 @@ public function setLastModified( $timestamp ) {
 	 * @return null
 	 */
 	public function setRobotPolicy( $policy ) {
-		$policy = Article::formatRobotPolicy( $policy );
 
+		header( "X-Robots-Tag: $policy" );
```

Comment:
I would consider avoiding doing this by default here, I think we should try to only use `X-Robots-Tag` when we can't output the meta HTML tag to start with. Additionally, it is technically possible for other code to change these values of the meta tag through `setIndexPolicy` and `setFollowPolicy` (the parser does for example), and it would be bad if this got out of sync. So, I'd consider limiting this to Special:Export when it outputs XML initially to limit testing scope and avoid unintended consequences.